README.md

```diff
@@ -27,14 +27,14 @@ Building REEF
 Requirements
 ------------
 
-  * Java 7 Development Kit
+  * Java 7 Development Kit or newer.
```

Comment:
Does REEF work with Java 8? I remember half a year ago when I was doing first-time workstation setup it didn't work with Java 8, I had to downgrade to 7, and that was expected behavior.